Makefile

```diff
@@ -19,6 +19,9 @@ GOTEST_INTEGRATION = $(GOTEST_BASE) -tags="forceposix integration"
 
 OS := $(shell uname)
 
+bin-clean:
```

Comment:
In order to follow `ci` api, I think we should override `ci/Makefile.main::clean` target, instead of using a new one.

Reply:
then it complains about overriding (warning). If everybody prefers that, I'm okay to change.

Reply:
yes, and imo the overriding warning is something expected when working with Make. And it's also a good way to notify that the used `clean` is different from the common one.

Reply:
I also prefer using `clean`.